src/plugins/intel_gpu/tests/test_cases/activation_simple_gpu_test.cpp

```diff
@@ -1630,14 +1637,38 @@ TEST_P(activation_random_test, random) {
     execute_compare(param, true);
 }
 
-INSTANTIATE_TEST_SUITE_P(activation_blocked_tests,
-                         activation_random_test,
-                         testing::ValuesIn(
-                            std::vector<activation_random_test_params>{
-                                { data_types::i8,  format::b_fs_yx_fsv32,        { 1, 32, 5, 5}, activation_func::relu, {}, {}},
-                                { data_types::i8,  format::bs_fs_yx_bsv32_fsv32, {32, 32, 5, 5}, activation_func::relu, {}, {}},
-                                { data_types::f16, format::bs_fs_yx_bsv32_fsv16, {32, 32, 5, 5}, activation_func::relu, {}, {}},
-                                { data_types::i8,  format::bs_fs_yx_bsv32_fsv32, {16, 16, 5, 5}, activation_func::relu, {}, {}},
-                                { data_types::f16, format::bs_fs_yx_bsv32_fsv16, {16, 16, 5, 5}, activation_func::relu, {}, {}},
-                            }
-                        ));
+const auto reluParams = testing::ValuesIn(std::vector<activation_random_test_params>{
+    {data_types::i8, format::b_fs_yx_fsv32, {1, 32, 5, 5}, activation_func::relu, {}, {}},
+    {data_types::i8, format::bs_fs_yx_bsv32_fsv32, {32, 32, 5, 5}, activation_func::relu, {}, {}},
+    {data_types::f16, format::bs_fs_yx_bsv32_fsv16, {32, 32, 5, 5}, activation_func::relu, {}, {}},
+    {data_types::i8, format::bs_fs_yx_bsv32_fsv32, {16, 16, 5, 5}, activation_func::relu, {}, {}},
+    {data_types::f16, format::bs_fs_yx_bsv32_fsv16, {16, 16, 5, 5}, activation_func::relu, {}, {}},
+});
+
+INSTANTIATE_TEST_SUITE_P(relu_activation_blocked_tests, activation_random_test, reluParams);
+
+const std::vector<data_types> dataTypes = {data_types::f16, data_types::f32};
+const std::vector<format::type> types = {format::bfyx,
+                                         format::bfzyx,
+                                         format::yxfb,
+                                         format::byxf,
+                                         format::fyxb,
+                                         format::b_fs_yx_fsv2,
+                                         format::b_fs_zyx_fsv2};
+const std::vector<activation_func> activationFunctions = {activation_func::gelu,
+                                                          activation_func::gelu_tanh,
+                                                          activation_func::softsign};
```

Comment:
can we add relu here also? woulld all the tests passed?

Reply:
Done